src/Board.php

```diff
@@ -97,4 +102,49 @@ public function jsonSerialize()
     {
         return json_encode($this->squares);
     }
+
+    public function ascii(): string
```

Comment:
It seems to me that this is not the board's responsibility. I would expect rather `AsciiPrinter` which implements:
```php
interface BoardPrinter
{
    public function print(Board $board): string
}
```

You can still work on the name itself. Examples: `Logger` or `Exporter`

Reply:
I agree.
I just would to move it away from main class in a quick (and maybe dirty) way.

Reply:
Added some classes to handle output (and squashed)